ansible/ansible.cfg

```diff
@@ -0,0 +1,3 @@
+[defaults]
```

Comment:
I'm personally not very familiar with ansible, so this file seems entirely magical to me both in existence, what the keys are, and what they're configured to. Could files perhaps have a link to a URL describing them and/or relevant documentation?